### PR TITLE
Kata week

### PR DESCRIPTION
**Pull request by yannick-cw**
* adds me as an author
* adds post about first week of katas at hc

### CODE REVIEW

File: _posts/2017-07-10-hc-katas.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Our goal for the first week was to refactor the code to small, readable methods and commit often and thereby improve our muscle memory for shortcuts and automated refactorings. The first two days we started out in small mobs or pairs, to discuss strategies of refactoring, working with *Scala* and *Javascript*.
+
+After being done with the first refactoring you might feel like continuing and improving more, but no - it's time to throw all progress away and start over again :).
+
+Soon we decided to switch to everyone working on their own machine, as the goal was to get as fast as possible.  
+The improvement for me was astonishing, I could refactor the whole code 5 times in the 45 minutes on Friday - the same refactoring that took me 45 minutes to do once on Wednesday.  
+Focussing more on shortcuts (spamming <kbd>command</kbd> + <kbd>p</kbd>, <kbd>command</kbd> + <kbd>m</kbd>. <kbd>command</kbd> + <kbd>v</kbd>) and not using the mouse anymore pushed my speed greatly.
+
+Besides all the fun and success we had in the first days, I got the impressions there are different views on what the coding kata is all about, so I'll try to list my opinion on that.
```
Comment (selaux) [inline _posts/2017-07-10-hc-katas.md]: `*impression`

File: _posts/2017-07-10-hc-katas.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+* discuss the results
+* **repeat**, repeat repeat
+* and **throw away**
+
+
+## What I think the goals of the kata are not
+
+* solving the given problem (*can* happen, must not)
+* reaching perfection
+* continuing with were you stoped yesterday
```
Comment (selaux) [inline _posts/2017-07-10-hc-katas.md]: `*stopped`

Comment (yannick-cw) [inline _posts/2017-07-10-hc-katas.md]: thanks, updated

**Comment by wolframkriesing** [comment]
pretty nice article

**Pull request merged**